### DISCUSSION TITLE
Budgie Roadmap ?

### DISCUSSION THREAD

**Discussion by greatbaracus** [BuddiesOfBudgie/budgie-desktop#238] [General]
Hi,
Merry Christmas to all.
Is there any roadmap of Budgie for the next months/years ?

There is not update since august this year with version 10.6.4, so i am wondering if the project is still active.

Thanks

**Comment by JoshStrobl** [comment]
Yes the project is still active. You can see commits happening and PRs being worked on. You can also see our roadmap info at https://github.com/orgs/BuddiesOfBudgie/projects and there will be a site that'll provide that info as well. See https://fosstodon.org/@JoshStrobl/109388156273289564

I would encourage you to follow the Mastodon account. Not saying it's a replacement for blog posts but it'll provide at least some info until I get around to finishing up some themeing and showing off everything coming in 10.7.
